zh-cn/yodaos-source/framework/07-hal.md

```diff
@@ -0,0 +1,6 @@
+# 硬件抽象接口
+
+[YodaOS][] 沿用了 Android Hardware Framework，这对于熟悉 Android 的开发者来说，可以更加轻松地切入到 [YodaOS][]
```

Comment:
I don't think its fully compatible with Android HAL.

Reply:
It should be compatible that I remembered, @zhubin0116 @Machaojoy maybe you guys could chime this in? But anyway this is ok, I decided to remove the hal section from this PR :)